andela/mmdp_site
# MMDP 799 Frequency not populated on stakeholder directory form

**What does this PR do?**

Ensure the frequency field is populated when editing an existing stakeholder's beneficiary information

**Description of Task to be completed?**

Ensure the frequency field is populated when editing an existing stakeholder's beneficiary information

**How should this be manually tested?**

- `git fetch`
- `git checkout bg-handle-frequency-not-populated-on-stakeholder-directory-form-MMDP-799`
- `yarn dev`
- Open `http://localhost:1234` on your browser and navigate to `http://localhost:1234/stakeholder-directory`
- Edit a stakeholder
- Ensure the frequency field is populated

**Any background context you want to provide?**

Currently, when one is updating stakeholder information, the option selected (when the stakeholder was created) from the "Frequency" dropdown does not appear on the beneficiary information form

**What are the relevant Jira issues?**

[MMDP-799](https://viisaus.atlassian.net/browse/MMDP-799)

**Screenshots (if appropriate)**
